### ISSUE TITLE
SB Theme Question

### ISSUE THREAD

**Issue by srd-software** [Alexey-T/ATButtons#30]
Hi Alexey,

Here's a question for you. When using the ATscrollbars, anytime we change something visual in the ATScrollBarTheme, we have to "manually" invalidate every atscrollbar on the form to get that change to show up. Can you think of a way to be able to send a message or something that would force all the form atscrollbars to update to a theme change at one time? I'm thinking maybe a private message (not sure how that would work in lazarus), or maybe a theme property of some sort that would react to a mass update call?

**Comment by Alexey-T** [comment]
In CudaText I just call myform.Invalidate, or maybe I call Invalidate for all visible controls which have scrollbars. It’s not ok?

**Comment by srd-software** [comment]
I tried a form invalidate in the scrollbar demo and that didn't change the atscrollbars. I still have to individually invalidate each one (although I guess I could put update code in the form paint method). 

I've got code in the editor I made that spins through the form components and updates the colors based on class when my theme changes. All that works, but I was just wondering if you knew of any better way to do that kind of mass update. If that's pretty much what you are doing then I'll stay with that.
